### DISCUSSION TITLE
各位，退款就是一直拖着，或者各种理由，不给退啊

### DISCUSSION THREAD

**Discussion by liuh23** [getlantern/lantern-forum-cn#1064] [General]
退款完全就是骗人的，你申请退款时会告诉你已经向上提交。
如果你多次催促，会让你去支付宝申请退款。
申请之后，你得到的商家回复是：”账户限制，无法退款，你需要跟支付宝客服申请退款“。
找了客服申请后，再次得到的商家回复是：”账户被限制了，要等180天账户解除限制才能退款“。
-------------------总之就是各种理由不给你退，一直拖着，照片为证。（如果你们诚心退款，那就直接支付宝，或者微信转账给我啊）

**Comment by liuh23** [comment]
详情见链接：https://github.com/getlantern/lantern-forum-cn/discussions/1014

**Comment by MaxMKMax** [comment]
@liuh23 我回复您的信息了. 退款的事情不是我弄的, 我只能帮您反馈. 蓝灯团队在美国, 用不了支付宝. 
来做转账. 但我能在帮您深入问清楚情况. 看还有什么办法吗. 非常抱歉啊.

**Comment by liuh23** [comment]
我看到了，我也给你回复了，就是因为你们给的方法无效，所以我才反馈。

**Comment by liuh23** [comment]
给我回复！！！一直拖着算怎么回事

**Comment by MaxMKMax** [comment]
@liuh23 您好. 但问题是我们的负责人已经和您说清楚我们这里没权限处理您的退款.  需要您向第三方那里 / 支付宝申请退款.

有话好好说. 你那么激动干嘛. 我都说了会尽力帮您. 你激动反而更难搞.

![image](https://github.com/user-attachments/assets/36b5e98b-8cc9-439e-9b03-af928189c986)

**Comment by MaxMKMax** [comment]
@liuh23 而且我们公司也没有支付宝. 你也懂得, 就算有也不能私底下随便转账, 美国哪里哪里可以直接给中国直接转账? 我都说了我会尽力帮你. 你等一等可以吗?

**Reply by liuh23** [reply]
可是我申请退款过后，收款方的回复是：账号有问题，退不了。
那我等你两个星期，12月24号能帮我处理好吗（从我申请退款到现在已经快两个月了）。
支付宝退不了的话，我可以把银行卡发你邮箱，用银行卡退

**Comment by MaxMKMax** [comment]
@greySkiesMarsh The user insist on getting refund. Can you help the user to do something? Can we have alternative options in refunding the users? Directly using Alipay (if we have), or bank transfer? 

I believe it would be great if you can handle the user and help him. Thanks.

cc: @liuh23
